config/jobs/kubernetes/sig-aws/sig-aws-credentials.yaml

```diff
@@ -0,0 +1,30 @@
+presets:
+# Credentials for using AWS test account 768319786644. Used for kops CNCF tests.
+- labels:
+    preset-aws-credential: "true"
+  env:
+  - name: AWS_SHARED_CREDENTIALS_FILE
+    value: /etc/aws-cred/credentials
```

Comment:
data should be /etc/aws-cred/aws-credentials?

Reply:
Yeah right.. Actually, since there's aws already in the path, can we rename the secret key to `credentials` instead of `aws-crendentials`? (sorry for asking to change again)